sdk/monitor/monitor-opentelemetry-exporter/samples-dev/basicTracerNode.ts

```diff
@@ -32,7 +25,13 @@ const exporter = new AzureMonitorTraceExporter({
     process.env["APPLICATIONINSIGHTS_CONNECTION_STRING"] ||
     "InstrumentationKey=00000000-0000-0000-0000-000000000000;",
 });
-provider.addSpanProcessor(new SimpleSpanProcessor(exporter as any));
+
+const provider = new BasicTracerProvider({
```

Comment:
need to run `npx dev-tool samples publish` to have this change included in published ts/js samples

Reply:
Fixed